Assets/Scripts/Entity/Player/Possession.cs

```diff
@@ -2,49 +2,95 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+
```

Comment:
What's with all these extra new lines? Trying to increase the lines of code you have commited? hu?

Reply:
lol guilty as charged. Jk not intentional

Reply:
What had happened was I was needed a namespace but ended up not needing it...